cas/ctg_store.c

```diff
@@ -2046,15 +2260,16 @@ int ctgdump(struct m0_motr *motr_ctx, char *fidstr, char *dump_in_hex_str)
 	for (rc = m0_be_btree_cursor_first_sync(&cursor); rc == 0;
 	     rc = m0_be_btree_cursor_next_sync(&cursor)) {
 		m0_be_btree_cursor_kv_get(&cursor, &key, &val);
-		memcpy(&out_fid, key.b_addr + M0_CAS_CTG_KV_HDR_SIZE,
+		memcpy(&out_fid, key.b_addr + M0_CAS_CTG_KEY_HDR_SIZE,
 		      sizeof(out_fid));
 		if (!m0_dix_fid_validate_cctg(&out_fid))
 			continue;
 		m0_dix_fid_convert_cctg2dix(&out_fid, &gfid);
 		M0_LOG(M0_DEBUG, "Found cfid="FID_F" gfid=:"FID_F" dfid="FID_F,
 				  FID_P(&out_fid), FID_P(&gfid), FID_P(&dfid));
 		if (m0_fid_eq(&gfid, &dfid)) {
-			ctg = *(struct m0_cas_ctg **)(val.b_addr + 8);
+			ctg = *(struct m0_cas_ctg **)(val.b_addr +
+						      M0_CAS_CTG_VAL_HDR_SIZE);
```

Comment:
Can we use a new data type _struct meta_value_ here?

Reply:
Yes, we can.
I was planning to update ctg_dump, in another new patch.
In this patch, I wanted to make it as it is.

(When I did the ctg_dump, I didn't know the existence of this patch.)

Reply:
Now, fixed.